cli/internal/cmd/iamcreate.go

```diff
@@ -235,10 +230,17 @@ func (c *iamCreator) create(ctx context.Context) error {
 		}
 	}
 
-	c.spinner.Start("Creating", false)
-
-	conf := createConfig(c.provider)
+	var conf config.Config
+	if flags.updateConfig {
+		c.cmd.Printf("The configuration file %q will be automatically updated and populated with the IAM values.\n", flags.configPath)
+		c.log.Debugf("Parsing config %s", flags.configPath)
+		if err = c.fileHandler.ReadYAML(flags.configPath, &conf); err != nil {
+			c.cmd.PrintErrf("There was an error reading the configuration file %q. Please make sure that this file exists and that you have the proper permissions for it.\n", flags.configPath)
+			return err
+		}
```

Comment:
I would agree. Not sure if you can be sure that "file not found" is the only error; so it may be better to wrap with something like `reading config file: %w`.